pandapipes/io/convert_format.py

```diff
@@ -42,6 +42,13 @@ def _rename_columns(net):
             else:
                 net['controller'].drop('controller', inplace=True, axis=1)
         net["controller"].rename(columns={"controller": "object"}, inplace=True)
+    for cp_tbl in ["circ_pump_mass", "circ_pump_pressure"]:
+        if cp_tbl in net:
+            for old_col, new_col in [("from_junction", "flow_junction"),
+                                     ("to_junction", "return_junction")]:
```

Comment:
Did we do that with other components, or was there a need to do something like that before? I don't really see a reason why we should not do that, and it might make life a little easier, that's true.

Reply:
Added this in the new commit.

Reply:
I don't think, that we did this before. It just came to my mind and as you also stated, it might make life easier. We should keep this in mind though, in case we stumble upon a similiar case.